### PR TITLE
Update Requirements

### PR DESCRIPTION
**Pull request by charliedrage**
Adds doc in coordination to #247 for all dependencies / requirements of Atomic App.

### CODE REVIEW

**Comment by vpavlin** [comment]
LGTM

File: docs/requirements.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Requirements
+
+Requirements needed to run AtomicApp.
```
Comment (goern) [inline docs/requirements.md]: it is 'Atomic App'

File: docs/requirements.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Python requirements
+
+* anymarkup >= 0.4.1
+* lockfile
+
+## Non-Python requirements
+
+__AtomicApp Core:__
```
Comment (goern) [inline docs/requirements.md]: Atomic App or atomicapp

File: docs/requirements.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+__Providers:__
+
+*  Docker Provider
+  * `docker` installed in the container (ver: 1.7.1, API ver: 1.19)
+* Kubernetes Provider
+  * `kubectl` either in container or in the /host (ver: v0.9.0)
+  * /etc/kubernetes either in container or in the /host
+* OpenShift Provider
+  * `oc` either in container oth in the /host
```
Comment (goern) [inline docs/requirements.md]: s/oth/or/

**Comment by charliedrage** [comment]
@goern updated

**Pull request merged**